src/main/kotlin/com/amazon/opendistroforelasticsearch/notifications/channel/email/BaseEmailChannel.kt

```diff
@@ -32,6 +32,11 @@ import javax.mail.internet.MimeMessage
  * Notification channel for sending mail to Email server.
  */
 internal abstract class BaseEmailChannel : NotificationChannel {
+
+    companion object {
+        private const val MINIMUM_EMAIL_HEADER_LENGTH = 160 // minimum of 100 emails
```

Comment:
np: I don't see how the comment `minimum of 100 emails` is related to this const declaration

Reply:
changed to "minimum value from 100 reference emails"